CONTRIBUTING.md

```diff
@@ -1,8 +1,23 @@
 # CLIMADA Contribution Guide
 
-Thank you for contributing to CLIMADA!
+We welcome any contribution to CLIMADA and want to express our thanks to everybody who contributes.
 
-## Overview
+## What Warrants a Contribution?
+
+Anything!
+For orientation, these are some categories of possible contributions we can think of:
+
+* **Technical problems and bugs:** Did you encounter a problem when using CLIMADA? Raise an [issue](https://github.com/CLIMADA-project/climada_python/issues) in our repository, providing a description or ideally a code replicating the error. Did you already find a solution to the problem? Please raise a pull request to help us resolve the issue!
+* **Documentation and Tutorial Updates:** Found a typo in the documentation? Is a tutorial lacking some information you find important? Simply fix a line, or add a paragraph. We are happy to incorporate you additions! Please raise a pull request!
+* **New Modules and Utility Functions:** Did you create a function or an entire module you find useful for your work? Maybe you are not the only one! As a module has to be carefully integrated into the framework, it might help if you talk to us first for designing and planning. You can do that by raising an issue or starting a [discussion](https://github.com/CLIMADA-project/climada_python/discussions) on GitHub.
```

Comment:
Here we could mention that there is a developer's call (does not need to be a direct link, just so people are aware that this is one way to get involved / joining a call may lower the barrier to contributing for some people).

Reply:
Great idea, see https://github.com/CLIMADA-project/climada_python/pull/797/commits/f241f0bbf63c3421017c5b55fd6fcbc321da8052